src/Component/component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './component.css';
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
```

Comment:
- [ ] Please know that the folder should be called `components` with a small c at the beginning and the file should be called `Calculator.js` with a capital C at the beginning, and the CSS file should match the component's file name, these are important conventions, kindly change that.